documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -259,10 +259,9 @@ Please consider using other types for the primary key when performing incrementa
 
 [NOTE]
 ====
-Incremental snapshots are currently supported for single replica set deployments only.
-ifdef::community[]
-This limitation will be removed in the next version.
-endif::community[]
+Incremental snapshots for sharded MongoDB clusters require the xref:mongodb-property-mongodb-connection-mode[`mongodb.connection.mode`] property set to `sharded`.
+
+For sharded MongoDB clusters the value of xref:mongodb-property-incremental-snapshot-chunk-size[`incremental.snapshot.chunk.size`] should be also configure sufficiently high in order to compensate for the link:https://www.mongodb.com/docs/manual/administration/change-streams-production-recommendations/#sharded-clusters[increased complexity] of change stream pipelines.
```

Comment:
```suggestion
To use incremental snapshots with sharded MongoDB clusters, you must set specific values for the following properties:

 * Set xref:mongodb-property-mongodb-connection-mode[`mongodb.connection.mode`] to `sharded`.

* Set xref:mongodb-property-incremental-snapshot-chunk-size[`incremental.snapshot.chunk.size`] to a value that is high enough to compensate for the link:https://www.mongodb.com/docs/manual/administration/change-streams-production-recommendations/#sharded-clusters[increased complexity] of change stream pipelines.
```
For this second item, I wonder whether we might be able to provide a general formula for calculating an adequately high value?